src/utilities.js

```diff
@@ -8,17 +7,15 @@ import _ from 'lodash';
  * @returns {Array}
  */
 const elementChildren = (element) => {
-  return _.filter(element.childNodes, {
-    nodeType: 1
-  });
+  return Array.from(element.childNodes).filter(elem => elem.nodeType === 1);
```

Comment:
Array.from isn't supported by IE11.

`element.childNodes` should be a DOMCollection, so I *think* you can do something like this
```js
Array.prototype.filter.call(element.childNodes, elem => elem.nodeType === 1)
```